### ISSUE TITLE
Editor doesn't allow image alt text to be set

### ISSUE THREAD

**Issue by pzeidman** [GitbookIO/feedback#493]
## Category

Feature request

## What is the current behavior?

The Editor doesn't provide an option to set an image's alt text. (Nor does it allow the raw markdown to be edited.) This means that the HTML which is generated is not compatible with accessibility laws (in the UK and probably elsewhere). Furthermore, plugins which make use of the image alt text to insert a caption cannot be used.

## What is the desired behavior?

When adding or editing an image, there should be a text box for changing the alt text, i.e.:

![alt text](http://www.google.com)

## Additional information

Applies to Web and Desktop.
